sky/clouds/aws.py

```diff
@@ -81,14 +103,8 @@ def region_zones_provision_loop(
     ) -> Iterator[Tuple[clouds.Region, List[clouds.Zone]]]:
         # AWS provisioner can handle batched requests, so yield all zones under
         # each region.
-        del accelerators  # unused
-
-        if instance_type is None:
-            # fallback to manually specified region/zones
-            regions = cls.regions()
-        else:
-            regions = service_catalog.get_region_zones_for_instance_type(
-                instance_type, use_spot, 'aws')
+        regions = cls.regions_with_offering(instance_type, accelerators,
+                                            use_spot, None, None)
```

Comment:
```suggestion
                                            use_spot, region=None, zone=None)
```
(styleguide) literals passed in as args are should use keywords to read easier

Reply:
Fixed.